### PR TITLE
Add assignment action set.

### PR DESCRIPTION
**Pull request by reiterl**
Add assignment action set.
Add tests.
Add some model parts.

### CODE REVIEW

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Adjust according to https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/60 and https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/61

**Comment by reiterl** [comment]
Update to new test base.

**Comment by reiterl** [comment]
Update the properties (create, update). Update the tests.
Notice: the two properties tag_ids and attachment_ids got problems so I excluded them.

**Comment by normanjaeckel** [comment]
@jsangmeister Please review.
@reiterl You may add the auto creation of agenda items according to paylod to the create action. See motion block and topic for details. Maybe also make some generic code if you like.

**Comment by reiterl** [comment]
Add auto creation of agenda item in the create action.

File: openslides_backend/action/assignment/create_update_delete.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            agenda_item_payload: ActionPayload = [agenda_item_payload_element]
+            agenda_item_creation.append((action, agenda_item_payload))
+
+        yield from super().create_write_request_elements(dataset)
+
+        for action, agenda_item_payload in agenda_item_creation:
+            yield from action.perform(agenda_item_payload, self.user_id)
+
+    def check_agenda_creation(self, flag: bool = None, meeting_id: int = None) -> bool:
+        # For assignments this should always return True.
```
Comment (normanjaeckel) [inline openslides_backend/action/assignment/create_update_delete.py:67]: No. For assignments the field agenda_creation (or something like this) should be evaluated. Please see motion_block

**Review by normanjaeckel — CHANGES_REQUESTED** [review]
Hey. You force-pushed away my changes ...

File: openslides_backend/action/assignment/create_update_delete.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/assignment/create_update_delete.py:67]: Sry, if I force pushed away your work. Are you working on my repo?

Comment (normanjaeckel) [reply openslides_backend/action/assignment/create_update_delete.py:67]: Of course. ;-)

If you like I can fix this branch an make everything ready. Please give me a ahort hint if this is ok for you.

Comment (reiterl) [reply openslides_backend/action/assignment/create_update_delete.py:67]: Okay, fix this branch.

**Comment by normanjaeckel** [comment]
@reiterl Please check my commit. Then you can rebase (don't forget to pull it ;-) and fix the last conflict. Then it is ready to merge. Thx.

File: openslides_backend/action/assignment/create_update_delete.py (hunk 1 above)
Comment (normanjaeckel) [reply openslides_backend/action/assignment/create_update_delete.py:67]: Oh sorry. This last point is not fixed. The function should not return always True but do some logic as in motion_block.

**Comment by jsangmeister** [comment]
> Notice: the two properties tag_ids and attachment_ids got problems so I excluded them.

What problems? Excluding properties won't fix them ;)

**Review by jsangmeister — CHANGES_REQUESTED** [review]
When you fix the `check_agenda_creation` and the tests accordingly, it can be merged.

**Comment by jsangmeister** [comment]
Rebased. @normanjaeckel please look into why tests are suddenly failing which have nothing to do with this PR.

File: openslides_backend/models/assignment.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    phase = fields.PositiveIntegerField(
+        description="The phase of this assignment.", enum=[1, 2, 3]
+    )
+    default_poll_description = fields.CharField(
+        description="The default_poll_description of this assignment."
+    )
+    number_poll_candidates = fields.BooleanField(
+        description="If assignment poll candidates are numbered."
+    )
+    agenda_item_id = fields.ForeignKeyField(
```
Comment (normanjaeckel) [inline openslides_backend/models/assignment.py:38]: This is a OneToOne field.

Comment (normanjaeckel) [reply openslides_backend/models/assignment.py:38]: This should fix the test (but almost all of them).

**Comment by reiterl** [comment]
Tried to fix the branch and rebase it, didn't work that well...

**Comment by normanjaeckel** [comment]
For the agenda creation thing see #155

**Pull request merged**